drv/lpc55-rng/src/main.rs

```diff
@@ -283,13 +284,22 @@ fn main() -> ! {
         )
     };
 
+    let mixer = mutable_statics! {
+        static mut MIXER: [Sha3_256; 1] = [Sha3_256::new; _];
```

Comment:
This is the change I had in mind -- I'm surprised it didn't reduce stack usage more! Oh well. It's still an improvement.